vendor/github.com/opencontainers/runc/libcontainer/cgroups/systemd/apply_systemd.go

```diff
@@ -296,17 +295,10 @@ func (m *Manager) Apply(pid int) error {
 		}
 	}
 
-	statusChan := make(chan string)
```

Comment:
@sjenning can you verify this is the end result we want? (this matches the release-3.9 branch... some of the reverts that made it in there didn't make it into master)

Reply:
yes, this looks correct in that it is the net effect of the patch, then the revert.